src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -600,40 +601,16 @@ private void advanceToNextActiveTripDate() throws URISyntaxException {
         }
```

Comment:
If we're not moving this to its own method, I think it could still use some work. Mainly I would like to see more comments peppering the while checks just to give more context to why there is a difference with the `plusDays` -- that's not immediately clear from the code. Also, the large comment block above could maybe use an edit for a simplified description of what's happening.

Reply:
Comments refactored in d95910c71758df3212e5f878a91a509f02839dd0.